source/index.txt

```diff
@@ -22,16 +16,60 @@ The Node driver documentation is organized as follows:
   /api-documentation
   /faq
   /issues-and-help
+  /compatibility
 
-Compatibility
--------------
 
-MongoDB Compatibility
-~~~~~~~~~~~~~~~~~~~~~
+Introduction
+------------
+The official MongoDB Node.js driver allows Node.js applications to connect and perform queries with MongoDB.
+
+.. note::
+   
+   These docs are for version 3.5 of the MongoDB Node.js driver. If
+   you're looking for an older version of the MongoDB Node.js driver
+   docs, `see the GitHub pages
+   <https://mongodb.github.io/node-mongodb-native/>`_. For the main
+   MongoDB documentation, see the :manual:`MongoDB Manual </>`.
+
+Quick Start
+-----------
+Learn how to quickly get a Node.js application working with MongoDB Atlas through the `Quick Start </quick-start/>`_ section
```

Comment:
- Who is this for? (beginners)
- What does "working" mean? We could be more descriptive here.
  - connect to a cluster
  - read and write data
  - whatever else the QS does
- "quickly" is still relative to the user. Maybe we could have some phrasing like "Jump right in with the quickstart to learn how to set up a basic connection to MongoDB Atlas and begin working with data."

Reply:
How about the alternative sentence: 

"Jump right in with the `Quick Start <link>`_ to learn how to establish a connection to MongoDB Atlas and begin to work with data"

Reply:
Fixed